docs/contributing/pull-request-guidelines/ci-checks.md

```diff
@@ -58,11 +58,15 @@ pre-commit run -a --config .pre-commit-config-optional.yaml
 
 ## spell-check-differential
 
-This workflow detects spelling mistakes using [CSpell](https://github.com/streetsidesoftware/cspell) with [our dictionary file](https://github.com/tier4/autoware-spell-check-dict/blob/main/.cspell.json).  
-You can submit pull requests to [tier4/autoware-spell-check-dict](https://github.com/tier4/autoware-spell-check-dict) to update the dictionary.
-
+This workflow detects spelling mistakes using [CSpell](https://github.com/streetsidesoftware/cspell) with [our dictionary file](https://github.com/tier4/autoware-spell-check-dict/blob/main/.cspell.json).
 Since it is difficult to detect errors with no false positives, it is an optional workflow, but it is preferable to remove spelling mistakes as many as possible.
 
+You have the following options if you need to use a word that is not registered in the dictionary.
+
+- If the word is only used in a few files, you can use [inline document settings "cspell:ignore"](https://cspell.org/configuration/document-settings/) to suppress the check.
+- If the word is widely used in the repository, you can create a local cspell json and pass it to the [spell-check action](https://github.com/autowarefoundation/autoware-github-actions/tree/main/spell-check).
+- If the word is common and may be used in many repositories, you can submit pull requests to [tier4/autoware-spell-check-dict](https://github.com/tier4/autoware-spell-check-dict) to update the dictionary.
```

Comment:
It is difficult to distinguish clearly, but [tier4/cspell-dicts](https://github.com/tier4/cspell-dicts) may be suitable for some words, so could you please add it?

Reply:
Added https://github.com/autowarefoundation/autoware-documentation/pull/481/commits/e11ea6595c8310ada2aa43653ec3708d98921a32.